requirements-examples.txt

```diff
@@ -8,4 +9,4 @@ nibabel >= 3.2.1
 numpy-stl >= 2.12.0
 trimesh >= 3.8.11
 torch >= 1.10.0
-monai >= 0.9.0
\ No newline at end of file
+monai == 0.9.0
```

Comment:
are there any specific issues working with monai v0.9.1? we can improve the core codebase...

Reply:
The v0.9.1 broke the use of invertd with the way it is used in the apps, I have created an issue to update the app and will consult the migration guide. Thanks.